package-lock.json

```diff
@@ -1,214 +1,99 @@
 {
   "name": "markdown-converter",
   "version": "5.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Please install at least npm v7, completely revert all changes made to this file (`git checkout package-lock.json origin/dev`) and re-run `npm install`.
Having this `package-lock.json` file generated using `npm` with a version < 7 breaks the CI process (as npm v6 and lower **doesn't** install the `peerDependencies` of `dependencies` automatically while npm v7 and higher does).

I assume this is the reason for the CI process to fail.

Reply:
Commit Resolved: *PR Modifications Package Lock Json* ( d28ede37d4e09345c9f30da013b047c0021f09cc )